managed/ui/src/components/tables/DeleteBackup/DeleteBackup.js

```diff
@@ -12,20 +12,29 @@ export default class DeleteBackup extends Component {
 
   confirmDeleteBackup = async () => {
     const {
-      tableInfo: { backupUUID },
+      tableInfo: {
+        backupUUID,
+        data,
+        type
+      },
       deleteBackup,
       onHide,
       onSubmit,
       onError
-    } = this.props;
+    } = this.props;   
+    let payload = [];
+
+    type === "bulkDelete" ? payload = data : payload.push(backupUUID);
```

Comment:
Why not do `const payload = type === 'bulkDelete' ? data : [backupUUID]`?

Reply:
Didn't think of it this way but implemented the suggested one.